framework/rbac/DbManager.php

```diff
@@ -600,6 +608,10 @@ public function getAssignment($roleName, $userId)
         if (empty($userId)) {
             return null;
         }
+        if ($this->cacheAssignments) {
```

Comment:
It does query assignments, just through `getAssignments()`. There's no extra query executed, `getAssigmnents()` would populate the query and the result would be returned immediately.